pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -533,6 +540,34 @@ RUN pip install --no-cache-dir -U ${SMD_MODEL_PARALLEL_URL}
 # Install SM Distributed DataParallel binary
 RUN SMDATAPARALLEL_PT=1 pip install --no-cache-dir ${SMD_DATA_PARALLEL_URL}
 
+# Install NCCL-tests
```

Comment:
Same for this - Can we add a small line in the form of the comment that states - "Needed for SM Health Checker"?

Reply:
Added